Add documentation for the new Weekly InstructLab CLI Meeting (#265)
instructlab/community: Collaboration.md

If you are new to the InstructLab project, the weekly community meeting is a great place to get started!

### [Weekly InstructLab CLI Meeting](#weekly-instructlab-cli-meeting)

> We host weekly InstructLab meetings each Thursday at 14:00 UTC. ([time zone converter](https://www.timeanddate.com/worldclock/meetingdetails.html?year=2024&month=5&day=14&hour=14&min=0&sec=0&p1=37&p2=43&p3=101&p4=224&p5=213&p6=771&p7=248&p8=2))

Reviewer: Propose rewording to "We host weekly InstructLab CLI meetings" as we are now running into the tool has the same name as the project and can cause confusion problem.

> Weekly InstructLab CLI meetings will feature announcements from the [CLI Maintainers](MAINTAINERS.md) team about the CLI project, overviews of the general roadmap the project is taking via our [Milestones](https://github.com/instructlab/instructlab/milestones) as well as the [Project Board](https://github.com/orgs/instructlab/projects/2), and dedicated time for members of community comment on and contribute to the direction of the project.

Reviewer 2: nit ... someday i'd like to turn on some kind of reasonable line length limit in markdown linting. When paragraphs are all on a single line, it makes diffs harder to read. Using an editor that can do the line wrapping automatically makes this not a big deal to enforce.

Author: Would not be opposed to that - especially in docs-heavy repos like `community` - but probably won't happen in this PR

Author: @lhawthorn @russellb opened https://github.com/instructlab/community/issues/268 to follow up on this

> Weekly InstructLab CLI meetings will feature announcements from the [CLI Maintainers](MAINTAINERS.md) team about the CLI project, overviews of the general roadmap the project is taking via our [Milestones](https://github.com/instructlab/instructlab/milestones) as well as the [Project Board](https://github.com/orgs/instructlab/projects/2), and dedicated time for members of community comment on and contribute to the direction of the project.

Reviewer 2: ```suggestion
Weekly InstructLab CLI meetings will feature announcements from the [CLI Maintainers](MAINTAINERS.md) team about the CLI project, overviews of the general roadmap the project is taking via our [Milestones](https://github.com/instructlab/instructlab/milestones) as well as the [Project Board](https://github.com/orgs/instructlab/projects/2), and dedicated time for discussing the direction of the project.
```
